service.subtitles.pipocas/resources/lib/pipocas.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: UTF-8 -*-
+# Copyright, 2020, HiGhLaNdeR, Leinad4Mind.
+# This program is distributed under the terms of the GNU General Public License, version 2.
+# http://www.gnu.org/licenses/gpl.txt
+
+
+import os
+from os.path import join as pjoin
+import sys
+import time
+import unicodedata
+import urllib
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcvfs
+import fnmatch
+try:
```

Comment:
please remove simplejson usage, from Helix and up you can simply `import json`

Reply:
you don't even import it as a dependency so it doesn't make much sense, will always throw.

Reply:
done.